draft-meynell-panrg-scion-research-questions.md

```diff
@@ -201,11 +201,36 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 * One AS belongs to many ISDs?
 
 
+## Beacon Forwarding Policy
+
+The idea behind "beaconing" is to discover all possible paths (loop free and with a fixed maximum length) between two CORE AS.
+Every AS forwards any received beacons to all neighbor CORE ASes unless this would cause a loop or exceed the fixed maximum length.
+Implemented naively, the number of paths (and beacons) grows exponentially with the network size.
+This is currently mitigated, primarily (and efficiently), by forwarding only the five "best PCBs" to neighbours.
+
+This "best PCB" practice has several implications:
+
+* It limits the number of available paths for endpoints such that only five truly diverse paths may be available to a given destination.
+  "truly diverse" means: Most ASes will receive five PCBs to a given destination from several of their neighbours. However, assuming
+  those ASes are use a similar PCB selection policy, it is not unlikely that each neighbor AS will forward a very
+  similar set of five PCB. In effect, there may be more than five paths available, but they are likely to diverge only on the first
+  one or two hops **(TBC: Is that true? There is no research confirming this hypothesis)**.
+  The problem that there may be only five truly diverse paths available is especially relevant for:
+  * Multipathing: endpoints may not be able to use more than five paths and no great selection of pats on offer.
+  * Unusual path policies, such as geofencing, may not be fulfillable by the limited number of paths on offer.
+* It causes a small number of paths to carry all traffic to a given destination, see {{link-load-balancing}}.
+* It opens the possibility of a DoS attack where a small number of "wormholes", see {{dos-with-wormholes}}.
```

Comment:
The possibility of attacking the selection of the best paths does not appear due to this beaconing process.
The same issue appears with an e.g. push-pull-publish-subscribe method, as long as the information propagated in the PCBs is trusted by the AS selecting the beacons and paths. Be that information latency, bandwidth, number of hops, etc (anything).